src/utils/safeNamehash.ts

```diff
@@ -0,0 +1,12 @@
+import { namehash } from '@ethersproject/hash'
+
+export function safeNamehash(name?: string): string | undefined {
+  if (typeof name === 'undefined') return undefined
+
+  try {
+    return namehash(name)
```

Comment:
Did you open an issue with @ethersproject? We should be linking a relevant issue here as a comment, so it can be cleaned up later if/when they resolve it.